CRM/Financial/BAO/FinancialTypeAccount.php

```diff
@@ -150,10 +150,11 @@ public static function del($financialTypeAccountId, $accountId = NULL) {
    *
    * @param int $paymentInstrumentValue
    *   Payment instrument value.
+   * @param bool $returnName
    *
    * @return null|int
    */
-  public static function getInstrumentFinancialAccount($paymentInstrumentValue) {
+  public static function getInstrumentFinancialAccount($paymentInstrumentValue, $returnName = FALSE) {
```

Comment:
Just as a point of style (& this is non-blocking) - I think it's more readable to use a wrapper function (e.g. getInstrumenetFinancialInstrumentName()) function than to start having a function return different formats in response to different input params